langserver/references.go

```diff
@@ -212,6 +212,32 @@ func (h *LangHandler) handleTextDocumentReferences(ctx context.Context, conn JSO
 		t := time.NewTicker(time.Second)
 		defer t.Stop()
 		streamTick = t.C
+		streamPos := 0
+		streamUpdate = func() {
+			mu.Lock()
+			partial := refs
+			mu.Unlock()
+			if len(partial) == streamPos {
```

Comment:
Maybe add a comment `// Everything in refs has already been sent, so return.` ? that would've answered my Q here :)